assemblyline_service_client/task_handler.py

```diff
@@ -365,6 +367,12 @@ def get_task(self) -> ServiceTask:
         return task
 
     def download_file(self, sha256, sid) -> Optional[str]:
+        if not SHA256_REGEX.match(sha256):
```

Comment:
SHA256_REGEX is a string, not a regex, so this will cause an AttributeError:
`AttributeError: 'str' object has no attribute 'match'`